File: src/main/java/pl/bk/contract/api/dto/ContractDto.java
package pl.bk.contract.api.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Getter
public class ContractDto
{
    private String countryISO;
    private String dailyGrossEarnings;
}

File: client/src/app/contract/contract-dto.ts
export class ContractDto {

  countryISO: String;
  dailyGrossEarnings: String;

  constructor(countryISO: String, dailyGrossEarnings: String) {
    this.countryISO = countryISO;
    this.dailyGrossEarnings = dailyGrossEarnings;
  }
}
